.github/workflows-off/slither.yml

```diff
@@ -0,0 +1,44 @@
+name: Slither Analysis
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    environment: CI
+    permissions:
+      contents: read
+      security-events: write
+    env:
+      NETWORK_TESTNET: "optimism-sepolia"
+      NETWORK_MAINNET: ${{ secrets.NETWORK_MAINNET }}
+      TESTNET_PRIVATE_KEY: ${{ secrets.TESTNET_PRIVATE_KEY }}
+      MAINNET_PRIVATE_KEY: ${{ secrets.MAINNET_PRIVATE_KEY }}
+      ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+      ETHERSCAN_BASE_SEPOLIA_API_KEY: ${{ secrets.ETHERSCAN_BASE_SEPOLIA_API_KEY }}
```

Comment:
should be the same apikey? I think ETHERSCAN_BASE_SEPOLIA_API_KEY is ETHERSCAN_API_KEY